web-local/README.md

```diff
@@ -0,0 +1,17 @@
+# web-local
+
+This folder contains the local frontend implemented with TypeScript and [SvelteKit](https://kit.svelte.dev). 
+
+## Running in development
+
+To run the web server with an embedded runtime:
+
+1. In `web-local/build-tools/postinstall_runtime.sh`, edit the `RUNTIME_VERSION` constant with the commit hash of the runtime version you want to use.
+2. Run `npm run dev`
+
+To run both the web server and an external development runtime:
```

Comment:
This doesn't quite have all instructions needed to run the runtime in dev mode.  Can we add those as well? AFAICT, this also entails
- installing Go
- installing Protobuf (which is not strictly required but better to have people install it here in case they do in fact need it later)
- running `npm install` after this

Reply:
Good catch - I added the `npm install` instruction.

And I added a "more resources" section to point to the relevant instructions that already exist in the codebase: `CONTRIBUTING.md` and `runtime/README.md`